### PR TITLE
Story1 display

### PR DESCRIPTION
**Pull request by PaulREnglish**
A pun for Charlie: 
Why did the students rebel?
Because they were being treated as objects in class!

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+{
+    $query = $db->prepare("SELECT `name`,`damage_per_second`, 
+       `rate_of_fire`, `ammo_consumption`, `ammo_efficiency`, 
+       `default_weapon_number`, `image_url` FROM `weapons`;");
+    $query->execute();
+
+    return $query->fetchAll();
+}
+
+///**
```
Comment (charliecog) [inline functions.php:19]: remove commented out code

File: index.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                </div>
+               <section>
+                   <?= displayWeaponData($weaponsData); ?>
+
+
+
+               </section>
+
+
+
```
Comment (charliecog) [inline index.php:48]: remove line breaks - 1 max

**Review by charliecog — COMMENTED** [review]
almost there - looking good

File: functions.php (hunk 1 above)
Comment (Zedonk1995) [reply functions.php:19]: done

File: index.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                </h1>
+            </header>
+            <main>
+                <div class="contents">
+                    <h2>
+                        Collection Contents
+                    </h2>
+
+                    <ul>
+                        <?= displayWeaponNames($weaponsData)?>
```
[the remark this answers is not in the document]
Comment (Zedonk1995) [reply index.php:37]: done

File: index.php [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                    </ul>
+                </div>
+               <section>
+                   <?= displayWeaponData($weaponsData); ?>
```
[the remark this answers is not in the document]
Comment (Zedonk1995) [reply index.php:41]: amended

File: index.php (hunk 2 above)
Comment (Zedonk1995) [reply index.php:48]: finished

File: tests/functions.php [hunk 5]
```diff
@@ -0,0 +1,168 @@
+<?php
+require_once'../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Functions extends TestCase
+{
+//    public function testSuccessValidateWeaponName()
```
[the remark this answers is not in the document]
Comment (Zedonk1995) [reply tests/functions.php:8]: removed

**Review by charliecog — APPROVED** [review]
looks good

**Pull request merged**